src/reconstruction/comparison/modelBorgifier/orderModelFieldsMB.m

```diff
@@ -1,10 +1,10 @@
-function Model = orderModelFields(Model)
+function Model = orderModelFieldsMB(Model)
```

Comment:
In PR https://github.com/opencobra/cobratoolbox/pull/914, the function `functionName` has been renamed as `functionBorgName`. May you do the same here?